lib/decoders/transform.js

```diff
@@ -1,3 +1,5 @@
+
+import _ from 'underscore';
```

Comment:
afaik lodash is a superset of underscore. i've just used underscore in the past since it's more minimal, but i guess that minimalism matters less when you aren't shipping the code to a browser...

could switch at some point, but no impetus yet really, other than that's what frontend uses.